app/assets/stylesheets/experiment/show.scss

```diff
@@ -224,3 +224,21 @@
     }
   }
 }
+
+.narrow {
```

Comment:
this needs to be scoped to an experiment-specific container, as the .narrow class is general-purpose and it should not have global styling